external/kafka-assembly/pom.xml

```diff
@@ -66,13 +66,16 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey</groupId>
-      <artifactId>jersey-server</artifactId>
-      <scope>provided</scope>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-client</artifactId>
```

Comment:
In fact, like the thrift server case, it may be the case that the explicit dependency isn't even needed. Just make sure the classes do not show up in the generated jar.